README.md

```diff
@@ -107,6 +107,13 @@ fuser = "0.15"
 
 To create a new filesystem, implement the trait `fuser::Filesystem`. See the [documentation] for details or the `examples` directory for some basic examples.
 
+### Feature Gates
+
+The crate uses feature gates to manage optional functionality and dependencies. Some key features include:
+*   **`abi-7-x`**: A set of features to select the FUSE protocol version. Recommended to select the highest version.
```

Comment:
We should recommend selecting the lowest version that provides the features the user needs. Not the highest. Selecting the highest will limit compatibility